website/partials/api.html

```diff
@@ -32,16 +32,17 @@ <h3 id="title">Protractor API <small>{{::version}}</small></h3>
         <span ng-repeat="item in items | filter:searchTerm"
             ng-switch="item.type">
             <div ng-switch-when="title"
-                class="title"
+                class="title {{item.treeClasses}}"
```

Comment:
Optional:
`class="title" ng-class="item.treeClasses"`

I'm not sure which I like better, up to you.